skipper.env

```diff
@@ -0,0 +1,42 @@
+CI
+VERBOSE
+SERVICE
+ISO_CREATION
+NAMESPACE
+PROFILE
+APPLY_NAMESPACE
+TARGET
+INGRESS_DOMAIN
+TEST_FLAGS
+FOCUS
+OCM_CLIENT_ID
+OCM_CLIENT_SECRET
+ENABLE_AUTH
+AUTH_TYPE
+SERVICE_REPLICAS_COUNT
+ASSISTED_TAG
+ASSISTED_ORG
+E2E_TESTS_CONFIG
+CHECK_CLUSTER_VERSION
+ENABLE_KUBE_API
+STORAGE
+CRD_OPTIONS
+TWINE_USERNAME
+TWINE_PASSWORD
+TWINE_REPOSITORY
+GENERATE_CRD
+BUNDLE_IMAGE
+INDEX_IMAGE
+DEBUG_SERVICE
+DEBUG_SERVICE_PORT
+LOCAL_ASSISTED_ORG
+DISABLED_HOST_VALIDATIONS
+CONTAINER_COMMAND
+DISABLED_STEPS
+MINIKUBE_HOME
+KUBECONFIG
+IMAGE
+VERSION
+REPO
+SPECIALRESOURCE
+IMG
```

Comment:
Do we have a way to get all the host's env variables mounted by default?

The way it is implemented now, for each new env variable to SRO we will need to update it here as well.

Reply:
currently no, but we don't expect to have many new env variables. In assisted-service, since everybody used skipper, so every added ENV variable had to be added to skipper.env. in SRO, we will have to add it manually

Reply:
So there will be no difference between skipper in assisted and skipper in SRO :( 
Anyway, I don't have any better solution, just was wondering if it is possible.